static/include/services/cartographer/configparams.md

```diff
@@ -1,5 +1,5 @@
 <!-- prettier-ignore -->
-| Parameter Mode | Description | Inclusion | Default Value | Notes |
+| Parameter Mode | Description | Required? | Default Value | Notes |
 | -------------- | ----------- | --------- | ------------- | ----- |
 | `mode` | `2d` | **Required** | None | |
```

Comment:
```suggestion
| `mode` | `2d` | **Required** | None | | |
```
I'm surprised this isn't throwing an error but should it be like this?

Reply:
no - 5 columns